images/metalk8s-utils/configure-repos.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+set -xue -o pipefail
+
+CENTOS_VERSION=$1
+SALT_VERSION=$2
+
+cat > /etc/yum.repos.d/kubernetes.repo << EOF
+[kubernetes]
+name=Kubernetes
+baseurl=https://packages.cloud.google.com/yum/repos/kubernetes-el$CENTOS_VERSION-x86_64
```

Comment:
Could have used the Yum vars, like `\$releasever` instead of this `CENTOS_VERSION` variable, no? And replaced `x86_64` with `\$basearch` maybe :)

Reply:
Hmh, yes, I guess... The 'source' (in `packages/redhat/repos/...`) from where I took this snippet doesn't use those either, I did add the `$CENTOS_VERSION`. But indeed, the 'default' variables would've been better. But merged now :man_shrugging: and not sure it's worth a follow-up :man_shrugging:

Reply:
Yeah not critical at all :ok_hand: